.github/workflows/preview.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       id-token: write
       pull-requests: write
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
Does this need Git LFS?

Reply:
I doubt it, as we're not installing any packages.